src/v/kafka/server/handlers/configs/config_response_utils.cc

```diff
@@ -0,0 +1,902 @@
+/*
```

Comment:
Is the `delete_retention_ms` vs `log_retention_ms` just a straight treewide rename? The merge diff is confusing because the string label is "log.retention.ms" in both cases.

Reply:
Yes, it looks like it was renamed here: https://github.com/redpanda-data/redpanda/pull/13257